vmm/src/device_manager.rs

```
@@ -4816,7 +4816,12 @@ impl Pausable for DeviceManager {
     fn pause(&mut self) -> result::Result<(), MigratableError> {
         for (_, device_node) in self.device_tree.lock().unwrap().iter() {
             if let Some(migratable) = &device_node.migratable {
-                migratable.lock().unwrap().pause()?;
+                match migratable.lock().unwrap().pause() {
+                    Err(e) if !matches!(e, MigratableError::UnSupported(_)) => {
+                        return Err(e);
+                    }
+                    _ => {}
```

This is wrong. Why are you ignoring the error type you just added?